app/src/layouts/common/projectSelector/projectSelector.jsx

```diff
@@ -70,18 +80,21 @@ export class ProjectSelector extends Component {
     }).isRequired,
   };
   static defaultProps = {
-    projects: [],
     activeProject: '',
     mobileOnly: false,
   };
 
   controlNode = null;
 
+  projects = [];
```

Comment:
May be rename it to `projectKeys` as it is an array of just keys, not the full objects.